JotunnLib/Managers/MockSystem/MockManager.cs

```diff
@@ -186,11 +209,24 @@ internal static void FixReferences(object objectToFix, int depth)
             }
         }
 
-        private static bool IsMockName(string name, out string cleanedName)
+        private static bool IsMockName(string name, out string cleanedName, out string pathName)
         {
+            pathName = null;
+
             if (name.StartsWith(JVLMockPrefix, StringComparison.Ordinal))
             {
-                cleanedName = name.Substring(JVLMockPrefix.Length);
+                int separator = name.IndexOf(JVLMockSeparator);
```

Comment:
```suggestion
                int separator = name.IndexOf(JVLMockSeparator, StringComparison.Ordinal);
```
using an Ordinal comparision is way faster and culture specifc strings are not relevant here